src/pages/member/subcomponents/Prizes.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Typography from '@material-ui/core/Typography'
+import Card from '@material-ui/core/Card'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+const styles = {
+    page: {
+        width: '719px',
+        marginLeft: '34px',
+        marginTop: '27px',
+    },
+    title: {
+        fontStyle: 'normal',
+        fontWeight: 'bold',
+        fontSize: '36px',
+        paddingLeft: '76px',
+        paddingTop: '45px'
+    }
+}
+
+function Prizes(props) {
```

Comment:
do we need 4 different components for each of these cards? Shouldn't they be pretty similar (and can all reuse the same component)

Reply:
i think the only thing that have in common is that they have a title and have react components but the react components are pretty different. Would it be cleaner to have 4 diff components or to have all the components created in UserHome.js and then passed to a helper component

Reply:
can make a general component in UserHome and have children passed to it https://stackoverflow.com/questions/49706823/what-is-this-props-children-and-when-you-should-use-it